src/display-pokemon-list.js

```diff
@@ -0,0 +1,273 @@
+              <div class="likes-box d-flex align-items-center ps-3"><span class="pe-2" id="${pokemon.index}_pokemonLikes">0</span>Likes</div>
+            </div>
+
+            <button type="button" class="Comments-button mt-3" data-bs-toggle="modal" data-bs-target="#popUpContainer${pokemon.index}" id="openComments">
+            Comments
+            </button>
+
+            <div class="modal fade rounded-3" tabindex="-1" aria-labelledby="popUpContainerLabel${pokemon.index}" aria-hidden="true" id="popUpContainer${pokemon.index}">
+            <div class="modal-dialog modal-xl rounded-3">
+              <div class="modal-content">
+                <div class="modal-header border-bottom border-dark border-5">
+                  <div class="d-flex">
+                    <div class="big-blue-circle me-3 mb-3"></div>
+                    <div class="red-circle me-2"></div>
+                    <div class="yellow-circle me-2"></div>
+                    <div class="green-circle me-2"></div>
+                  </div>
+                  <div class="d-flex">
+                    <button type="button" class="btn close me-3" data-bs-dismiss="modal">
+                      <ion-icon class="fs-close" name="close"></ion-icon>
+                    </button>
+                  </div>
+                </div>
+                <div class="modal-body">
+                  <div class="d-flex justify-content-center">
+                    <div class="gray-box">
+                      <div class="d-flex justify-content-center">
+                        <div class="small-red-circle m-1"></div>
+                        <div class="small-red-circle m-1"></div>
+                      </div>
+                      <div class="d-flex justify-content-center">
+                        <div class=" display-green d-flex justify-content-center align-items-center" id="${pokemon.index}PokemonImageComment">
+                        </div>
+                      </div>
+                      <div class="d-flex justify-content-between px-4">
+                        <div class="medium-red-circle m-1"></div>
+                        <ion-icon name="menu-sharp" class="fs-1"></ion-icon>
+                      </div>
+                    </div>
+                  </div>
+                  <div>
+                    <h3 class="add-comment-heading">Add a comment</h3>
+                    <div id="commentsForm">
+
+                    <form class="form" id="form" onsubmit="return false">
+                      <input type="hidden" name="pokemonId" value="${pokemon.index}" id="pokemonId">
+                      <input type="text" placeholder="Your name" id="username" />
+                      <textarea name="comment" id="comment" cols="20" rows="10" placeholder="Your insights"></textarea>
+                      <input type="submit" value="Comment" class="btn btn-dark disabled" id="submitComment">
+                    </form>
+
```

Comment:
- [ ] OPTIONAL: I think it is better if the input field is cleared after form submission. You can make this change when you get spare time. 

![image](https://user-images.githubusercontent.com/64894927/132952841-0796bc75-24ea-4bcc-899d-80e2ba1b8385.png)

Reply:
Thank you very much for your recommendations, we will certainly make them, I hope you have a happy weekend.

Reply:
Now I fix this problem, thanks for your recomendation